src/helpers/utils.js

```diff
@@ -32,7 +32,10 @@ export default {
    * @returns {string} - return jwt token
    */
   jwtSigner(payload) {
-    return jwt.sign(payload, process.env.JWT_SECRET, { expiresIn: '24h' });
+    if (!payload.time) {
+      return jwt.sign(payload, process.env.JWT_SECRET);
```

Comment:
You forgot to set the expiresIn to be 24hrs

Reply:
THE TIME IS SET AT THE PLACE USED BY THIS METHOD AND I AM NOT USING 24HR TIME

Reply:
I meant while refactoring this method you left out the expiresIn 24h. Compare the line removed with what is on line 36.

Reply:
change the jwtSigner to this:

```
jwtSigner(payload, expiresIn='24h') {
  return jwt.sign(payload, process.env.JWT_SECRET, { expiresIn });
} 
```

usage:
```
jwtSigner(somePaylod) // expiration time will be 24hrs (default)

jwtSigner(anotherPaylod, '1hr') // expiration time will be 1hr (overrides the default)
```